utility/src/main/java/org/oppia/android/util/parser/html/CustomHtmlContentHandler.kt

```diff
@@ -185,6 +187,21 @@ class CustomHtmlContentHandler private constructor(
      * @param indentation The zero-based indentation level of this item.
      */
     fun handleClosingTag(output: Editable, indentation: Int, tag: String) {}
+
+    /**
+     * Called when a custom tag is encountered. This is always called after the closing tag.
```

Comment:
This documentation doesn't actually seem correct per what this function is used for--please update.